src/Product.php

```diff
@@ -9,6 +9,8 @@ class Product {
 
   const CATEGORY_EXCLUDED = ['Ausstellungsstücke', 'Abverkauf'];
 
+  private static $allVariationsStock;
+
```

Comment:
```suggestion
  private static $allVariationsStock;
```

Can you add documentation for this member variable?

Reply:
Updated in the last commit.